src/components/Link/index.js

```diff
@@ -7,24 +7,18 @@ import StyledLink from './styles/StyledLink'
 /**
  * A link to an action or another resource.
  */
-class Link extends PureComponent {
-  render() {
-    const { children, className, href, onClick, target, ...rest } = this.props
-
-    return (
-      <StyledLink
-        className={className}
-        href={href}
-        onClick={onClick}
-        target={target}
-        rel={target === '_blank' ? 'noreferrer noopener' : ''}
-        {...rest}
-      >
-        {children}
-      </StyledLink>
-    )
-  }
-}
+const Link = ({ children, className, href, onClick, target, ...rest }) => (
+  <StyledLink
+    className={className}
+    href={href}
+    onClick={onClick}
+    target={target}
+    rel={target === '_blank' ? 'noreferrer noopener' : ''}
+    {...rest}
```

Comment:
Using `React.memo()` would be justified here too in my opinion, because of the `{...rest}` prop passed

Reply:
Same remarks I did for Input, in my opinion, the components should be optimized on user side, not here, because you can't know what is in `rest`